config/grafana/overview.json

```diff
@@ -1994,5 +1995,5 @@
   "timezone": "",
   "title": "Lifecycle Manager Overview",
   "uid": "O3DH7uunk",
-  "version": 7
-}
+  "version": 1
```

Comment:
is this version correct? It was 7

Reply:
Was set by Plutono itself. I just changed the two lines with adding the container name. The rest was done by plutono

Reply:
So I guess, it should be correct

Reply:
Yeah I was just surprised that it was downgraded so was just making sure you didn't edit it by mistake